extensions-core/google-extensions/src/test/java/org/apache/druid/storage/google/GoogleTaskLogsTest.java

```diff
@@ -134,19 +134,21 @@ public void testStreamTaskLogWithoutOffset() throws Exception
   public void testStreamTaskLogWithPositiveOffset() throws Exception
   {
     final String testLog = "hello this is a log";
+    final String expectedLog = testLog.substring(5);
```

Comment:
Since it appears in a couple places for this method, may be useful to have a named constant in this method for the value 5, similar to what was done in `testStreamTaskLogWithNegative()`.